### PR TITLE
add reference to permissions plugin in order to request permission for using the camera

### PR DESCRIPTION
**Pull request by MaikuB**

### CODE REVIEW

File: src/BuildIt.Forms/Samples/BuildIt.forms.Sample.Core/ViewModels/MainViewModel.cs [hunk 1]
```diff
@@ -114,11 +116,21 @@ public void SwitchStates()
                 await item.Init();
                 items.Add(item);
             }
+
             Items.Fill(items);
             MoreItems.Fill(items);
+            // need to request runtime permissions for using the camera
+            var results = await CrossPermissions.Current.RequestPermissionsAsync(Permission.Camera);
+            if (results.ContainsKey(Permission.Camera))
```
Comment (nickrandolph) [inline src/BuildIt.Forms/Samples/BuildIt.forms.Sample.Core/ViewModels/MainViewModel.cs]: I think BuildIt.General has an extension to get value, or default if it doesn't exist

Comment (MaikuB) [reply src/BuildIt.Forms/Samples/BuildIt.forms.Sample.Core/ViewModels/MainViewModel.cs]: Yep though requires casting results to IDictionary and the dictionary has enum values. Letting it return the default (which I think would be saying the permission was denied) could be a bit misleading/inaccurate

**Comment by duckwaffle** [comment]
There should be a task in TFS for fixing the requirement to cast to IDictionary ;)

**Comment by nickrandolph** [comment]
Log a bug with the permissions plugin - it should return IDictionary, not Dictionary ;-)

**Comment by MaikuB** [comment]
@duckwaffle you did say you'd most likely be submitting PRs still so...do it!

**Pull request merged**